CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ And please only add new entries to the top of this list, right below the `# Unre
 
 # Unreleased
 
+- **Breaking:** On Windows, add support for two-finger touchpad magnification gesture with new event `WindowEvent::TouchpadMagnify`.
```

Comment:
Can you explain here why it's a breaking change (maybe separate the changelog entry in two?)

Reply:
I was thinking that maybe some people have already implemented the magnification gesture in their code by checking if the control key is activated when they receive a MouseWheel event. But with these changes, their code might not work on Windows anymore because MouseWheel will no longer be emitted. 

And I also follow this #2157 which is also a note as a break change.